packages/lib-classifier/src/plugins/tasks/experimental/volumetric/components/VolumetricTask.spec.js

```diff
@@ -0,0 +1,24 @@
+import { composeStory } from "@storybook/react"
+import { render, screen } from "@testing-library/react"
+import Meta, { Default } from "./VolumetricTask.stories"
+
+describe("VolumetricTask", function () {
+  const DefaultStory = composeStory(Default, Meta)
+
+  describe("when it renders", function () {
+    it("should show the instruction", function () {
+      render(<DefaultStory />)
+      expect(screen.getByText("Volumetric the task")).to.be.ok()
+    })
+
+    it("should show the label text", function () {
+      render(<DefaultStory />)
+      expect(screen.getByText("3D Viewer")).to.be.ok()
+    })
+
+    it("should show the classification count", function () {
+      render(<DefaultStory />)
+      expect(screen.getByText("InputStatus.drawn")).to.be.ok()
+    })
```

Comment:
This works for now, but `InputStatus.drawn` also includes the variable `count`. A more robust test would look for `count = 3` (because that's what you've hardcoded it as for now), and when the annotations feature is added for volumetric subjects, then you'd pass `annotations` to the VolumetricTask stories and look for the appropriate count number used in the stories.

Reply:
Yup - agreed. That'll come in the next PR.